app/vue/src/client/preview/index.js

```diff
@@ -19,6 +19,10 @@ const decorateStory = (getStory, decorators) =>
         decoratedStory = { template: decoratedStory };
       }
 
+      if (decorator.name === 'withSubscriptionTracking') {
```

Comment:
If this solved a decorators issue, maybe it should be done for all the frameworks?

Reply:
I think its related to how we create and wrap functional components in Vue specifically. I looked at snapshot tests in angular and react and it seems the issue doesn't occur there.